spec/std/array_spec.cr

```diff
@@ -144,6 +144,16 @@ describe "Array" do
     it "does with even larger arrays" do
       ((1..64).to_a - (1..32).to_a).should eq((33..64).to_a)
     end
+
+    context "with different types" do
+      it "small array)" do
+        ([1, 2, 3, 'c'] - [2, nil]).should eq [1, 3]
+      end
+
+      it "big array)" do
+        (((1..64).to_a + 'c') - ((2..63).to_a + [nil])).should eq [1, 64]
```

Comment:
This doesn't compile. Did you really push the wrong commit?

Reply:
Oops, there was some manual trying-to-be polishing up just before pushing 😆

Reply:
If we no longer target old versions on CI, this could also be written as `[*(1..64), 'c'] - [*(2..63), nil]`